plugins/modules/identity/sap_company.py

```diff
@@ -10,21 +10,20 @@
 ---
 module: sap_company
 
-short_description: This module will manage a company entities in a SAP S4HANA environment.
+short_description: This module will manage a company entities in a SAP S4HANA environment
 
-version_added: "0.2.0"
+version_added: "1.0.0"
```

Comment:
```suggestion
version_added: "1.0.0"
```
As i don't see any versions older than 0.1.0. If it's about community.general, we should also use `version_added_collection`